packages/testing-toolkit/src/index.ts

```diff
@@ -0,0 +1,28 @@
+import { fireEvent, screen } from '@testing-library/react';
+
+export async function inputText(canvasElement: any, linkId: string, mytext: string) {
+  const questionElement = await canvasElement.querySelector(`[data-linkid=${linkId}]`);
+  const input = questionElement.querySelector('input') ?? questionElement.querySelector('textarea');
+
+  if (!input)
+    throw new Error(`Input or textarea was not found inside ${`[data-linkid=${linkId}] block`}`);
+
+  fireEvent.change(input, { target: { value: mytext } });
+}
+
+export async function chooseSelectOption(canvasElement: any, linkId: string, optionLabel: string) {
+  const questionElement = await canvasElement.querySelector(`[data-linkid=${linkId}]`);
+  const input = questionElement.querySelector('input');
+
+  if (!input) throw new Error(`There is no input inside [data-linkid=${linkId}] block`);
+
+  fireEvent.focus(input);
+  fireEvent.keyDown(input, { key: 'ArrowDown', code: 40 });
+
+  const option = await screen.findByText(optionLabel);
```

Comment:
have you tried to stick to canvas instead of screen? I suppose screen is too global, and we're working with specific canvas